model/Core/Vocabularies/HashAlgorithm.md

```diff
@@ -18,24 +18,24 @@ practically infeasible to invert.
 
 ## Entries
 
-- adler32: Adler-32 checksum is part of the widely used zlib compression library as defined in [RFC 1950](https://www.rfc-editor.org/info/rfc1950) Section 2.3.
-- blake2b256: BLAKE2b algorithm with a digest size of 256, as defined in [RFC 7693](https://www.rfc-editor.org/info/rfc7693) Section 4.
-- blake2b384: BLAKE2b algorithm with a digest size of 384, as defined in [RFC 7693](https://www.rfc-editor.org/info/rfc7693) Section 4.
-- blake2b512: BLAKE2b algorithm with a digest size of 512, as defined in [RFC 7693](https://www.rfc-editor.org/info/rfc7693) Section 4.
+- adler32: Adler-32 checksum is part of the widely used zlib compression library as defined in [RFC 1950](https://datatracker.ietf.org/doc/rfc1950) Section 2.3.
+- blake2b256: BLAKE2b algorithm with a digest size of 256, as defined in [RFC 7693](https://datatracker.ietf.org/doc/rfc7693) Section 4.
+- blake2b384: BLAKE2b algorithm with a digest size of 384, as defined in [RFC 7693](https://datatracker.ietf.org/doc/rfc7693) Section 4.
+- blake2b512: BLAKE2b algorithm with a digest size of 512, as defined in [RFC 7693](https://datatracker.ietf.org/doc/rfc7693) Section 4.
 - blake3: [BLAKE3](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf)
 - crystalsDilithium: [Dilithium](https://pq-crystals.org/dilithium/)
 - crystalsKyber: [Kyber](https://pq-crystals.org/kyber/)
 - falcon: [FALCON](https://falcon-sign.info/falcon.pdf)
-- md2: MD2 message-digest algorithm, as defined in [RFC 1319](https://www.rfc-editor.org/info/rfc1319/).
-- md4: MD4 message-digest algorithm, as defined in [RFC 1186](https://www.rfc-editor.org/info/rfc1186).
-- md5: MD5 message-digest algorithm, as defined in [RFC 1321](https://www.rfc-editor.org/info/rfc1321).
+- md2: MD2 message-digest algorithm, as defined in [RFC 1319](https://datatracker.ietf.org/doc/rfc1319/).
```

Comment:
Minor - while we're editing, let's remove the trailing `/` for consistency (the link works either way)

Reply:
Neat. Thank you.

Reply:
I have checked, the canonical URL has the /.
Will make all the same.

Reply:
Changed all to https://datatracker.ietf.org/doc/rfcXXXX/ (with trailing / )

Notes:
- https://www.rfc-editor.org/info/rfc3986 is without trailing / (RFC's own Bibtex use this format and the URL will not work with the trailing / )
- on the opposite, https://datatracker.ietf.org/doc/rfc3986/ is its canonical URL (the version without will redirected to the version with)